### PR TITLE
#161778452 Hide bed space column when trip is oneWay

### PR DESCRIPTION
**Pull request by ezrqnkemboi**
#### What does this PR do?
- Hide bed space selection when trip is oneWay

#### Description of Task to be completed?
On travel request initiation for a one-way trip, bed space selection field should not visible to the traveller.

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- git checkout ch-hide-bedspace-one-way-161778452
- yarn start 
- Run the server/backend on develop branch
- Login and make a new request (oneWay, multi and return)

#### Any background context you want to provide?
- In the oneWay trip, you will realize that bed space selection is not available while in multi or return, bed space selection is available.

#### What are the relevant pivotal tracker stories?
[#161778452] (https://www.pivotaltracker.com/story/show/161778452)

#### Screenshots (if appropriate)
screen shot 2018-11-12 at 10 49 43

#### Questions:

### CODE REVIEW

**Comment by amarachukwu-agbo** [comment]
Please include test for this fix and also fix the coverage drop

**Comment by ezrqnkemboi** [comment]
I am looking at that @amarachukwu-agbo . Thank you.

File: src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js [hunk 1]
```diff
@@ -4,54 +4,69 @@ import TravelDetails from '../TravelDetails';
...
-    it('should render properly if selection is oneWay', () => {
-        const newProps = {...props};
-        newProps.selection = 'oneWay';
-        delete newProps.values['arrivalDate-1'];
-        const wrapper = setup(props);
-        expect(wrapper.length).toBe(1);
-    });
-})
\ No newline at end of file
+  it('should render properly if selection is oneWay', () => {
```
Comment (amarachukwu-agbo) [inline src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js]: @ezrqnkemboi , we should be checking that the div that contains the available room input is no in the DOM when the user selects oneWay

Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js]: Why should I check that and what I am passing does not contain available room. Maybe the logic of actually accepting calling of available rooms be removed.  It does not make any sense to have such function and that is what can make chances of having available beds in DOM.

**Comment by Spectrumsun** [comment]
@ezrqnkemboi  what is keep this task. we need to work on another task

File: src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js (hunk 1 above)
Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js]: Writing the test for that.

**Comment by ezrqnkemboi** [comment]
Alright @Spectrumsun , working to fulfill that.

File: src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js (hunk 1 above)
Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/__tests__/TravelDetails.test.js]: I have made the changes.

**Pull request merged**